qgrid/grid.py

```diff
@@ -372,6 +446,7 @@ def __init__(self, *args, **kwargs):
         # register a callback for custom messages
         self.on_msg(self._handle_qgrid_msg)
         self._initialized = True
```

Comment:
It's because the traitlets `_changed` callbacks get called when we call the constructor of the super class `super(QgridWidget, self).__init__(*args, **kwargs)`.  When that happens we don't want to rebuild the widget like we normally do when people change those values.